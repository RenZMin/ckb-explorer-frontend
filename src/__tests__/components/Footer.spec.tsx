import { ReactElement } from 'react'
import renderer from 'react-test-renderer'
import Footer from '../../components/Footer'

describe('Footer Component', () => {
  let component: ReactElement

  beforeAll(() => {
    component = <Footer />
  })

  it('shallow renders', () => {
    const wrapper = renderer.create(component).toJSON()
    expect(wrapper).toMatchSnapshot()
  })
})
