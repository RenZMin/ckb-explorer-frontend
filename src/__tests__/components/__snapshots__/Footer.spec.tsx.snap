// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component shallow renders 1`] = `
<div
  className="sc-AxjAm grmmOl"
>
  <div
    className="sc-AxirZ RJiDd"
  >
    <div
      className="footer__foundation"
    >
      <div
        className="footer__title"
      >
        Foundation
      </div>
      <a
        className="sc-AxiKw dTWoaG"
        href="https://www.nervos.org/"
        rel="noopener noreferrer"
        target="_blank"
      >
        About Us
      </a>
    </div>
    <div
      className="footer__developer"
    >
      <div
        className="footer__title"
      >
        Developer
      </div>
      <a
        className="sc-AxiKw dTWoaG"
        href="https://docs.nervos.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        Docs
      </a>
      <a
        className="sc-AxiKw dTWoaG"
        href="https://github.com/nervosnetwork"
        rel="noopener noreferrer"
        target="_blank"
      >
        GitHub
      </a>
      <a
        className="sc-AxiKw dTWoaG"
        href="https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0002-ckb/0002-ckb.md"
        rel="noopener noreferrer"
        target="_blank"
      >
        Whitepaper
      </a>
      <a
        className="sc-AxiKw dTWoaG"
        href="https://faucet.nervos.org/"
        rel="noopener noreferrer"
        target="_blank"
      >
        Faucet
      </a>
      <a
        className="sc-AxiKw dTWoaG"
        href="mailto:asset-info-submit@nervina.io?subject=Submit Token Info&body=
Title: Submit Token Information%0a%0d
Type Script:%0a%0d
    Code Hash:%0a%0d
    Hash Type:%0a%0d
    Args:%0a%0d

Information:%0a%0d
   Full Name: 32 length max%0a%0d
   Symbol: 8 length max / ASCII%0a%0d
   Decimal: 8 (default)%0a%0d
   Description:%0a%0d
   Website:%0a%0d
   Icon File: attachment (40 x 40)%0a%0d
   Other Info:%0a%0d

Note: Only accept sUDT information submission now.
"
        rel="noopener noreferrer"
        target="_blank"
      >
        Submit Token Info
      </a>
    </div>
    <div
      className="footer__community"
    >
      <div>
        <a
          className="sc-AxhCb dindIV"
          href="https://twitter.com/nervosnetwork"
          onMouseLeave={[Function]}
          onMouseOver={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="footer icon"
            src="footer_twitter.png"
          />
          <span>
            Twitter
          </span>
        </a>
        <a
          className="sc-AxhCb dindIV"
          href="https://medium.com/nervosnetwork"
          onMouseLeave={[Function]}
          onMouseOver={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="footer icon"
            src="footer_medium.png"
          />
          <span>
            Blog
          </span>
        </a>
        <a
          className="sc-AxhCb dindIV"
          href="https://t.me/nervosnetwork"
          onMouseLeave={[Function]}
          onMouseOver={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="footer icon"
            src="footer_telegram.png"
          />
          <span>
            Telegram
          </span>
        </a>
      </div>
      <div>
        <a
          className="sc-AxhCb dindIV"
          href="https://www.reddit.com/r/NervosNetwork/"
          onMouseLeave={[Function]}
          onMouseOver={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="footer icon"
            src="footer_reddit.png"
          />
          <span>
            Reddit
          </span>
        </a>
        <a
          className="sc-AxhCb dindIV"
          href="https://www.youtube.com/channel/UCONuJGdMzUY0Y6jrPBOzH7A"
          onMouseLeave={[Function]}
          onMouseOver={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="footer icon"
            src="footer_youtube.png"
          />
          <span>
            YouTube
          </span>
        </a>
        <a
          className="sc-AxhCb dindIV"
          href="https://talk.nervos.org/"
          onMouseLeave={[Function]}
          onMouseOver={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="footer icon"
            src="footer_forum.png"
          />
          <span>
            Forum
          </span>
        </a>
      </div>
    </div>
  </div>
  <div
    className="footer__copyright"
  >
    <span>
      Copyright Â© 2021 Nervos Foundation. 
    </span>
    <span>
      All Rights Reserved.
    </span>
  </div>
</div>
`;
